test/models/badges/functional_february_badge_test.rb

```diff
@@ -0,0 +1,52 @@
+require "test_helper"
+
+class Badges::FunctionalFebruaryBadgeTest < ActiveSupport::TestCase
+  test "attributes" do
+    badge = create :functional_february_badge
+    assert_equal "Functional February", badge.name
+    assert_equal :ultimate, badge.rarity
+    assert_equal :functional, badge.icon
+    assert_equal 'Completed five exercises in a functional language in February', badge.description
```

Comment:
I wonder if we should make it clear that it is only for functional languages featured in the Functional February theme (e.g. LISPS are functional but don't count)

```suggestion
    assert_equal 'Completed five exercises in a functional language in February', badge.description
```

Reply:
Yeah. We have no more characters spare though :S